src/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Footer = props => {
+  return (
+    <div id="Footer" className="Footer_CSS">
+      <p className="Footer_P_CSS">Email Address: {props.Info[1]}</p>
+      <p className="Footer_P_CSS">Tell: {props.Info[2]}</p>
+      <p className="Footer_P_CSS">Address: {props.Info[0]}</p>
```

Comment:
This works ok, but exercise 4 asks you to use the `map` method to render a list. Could you update this component to use that instead?

Reply:
why I didn't use map because I added label before each item , but I'll change it to map. thank you